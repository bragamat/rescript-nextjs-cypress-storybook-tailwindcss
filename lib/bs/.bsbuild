8
Home
Next
_App
Navber
Cypress
Default
Storybook
_Document
src/Pages/Home	bindings/Next	src/App	src/Components/Navbar	bindings/Cypress	src/Layouts	bindings/Storybook	
1	
0
