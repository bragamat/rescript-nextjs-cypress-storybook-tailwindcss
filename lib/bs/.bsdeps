10.1.3
/Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs
0
bsconfig.json	0x1.902a23p+30
bindings	0x1.902a0e7p+30
bindings/Cypress	0x1.902a2014p+30
bindings/Storybook	0x1.902a2014p+30
bindings/Next	0x1.902a2014p+30
src	0x1.902a20ecp+30
src/Pages	0x1.902a20ecp+30
src/Pages/Home	0x1.902a2014p+30
src/Layouts	0x1.902a2014p+30
src/Components	0x1.902a106cp+30
src/Components/Navbar	0x1.902a2014p+30
src/App	0x1.902a217p+30
===
/Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/rescript/darwin/rescript.exe	0x1.902a2318p+30
