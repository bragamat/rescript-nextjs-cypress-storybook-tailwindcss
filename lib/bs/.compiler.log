#Start(1678411974287)
#Done(1678411974704)
