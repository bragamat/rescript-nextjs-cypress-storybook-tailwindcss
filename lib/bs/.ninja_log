# ninja log v6
1	52	1678411781530206192	bindings/Cypress/Cypress.ast	2926af30cb5c639e
2	76	1678411781553418189	bindings/Storybook/Storybook.ast	b33e67b1bb9b6f2b
5	78	1678411781554944264	src/Pages/App/_App.iast	ff5205b7809dcf2f
4	81	1678411781559981353	src/Pages/App/_App.ast	38a10329f90d5a16
3	86	1678411781564711302	bindings/Next/Next.ast	be00fa04f77b8d89
7	94	1678411781572310264	src/Pages/App/_Document.iast	9b7fc802b45f7d9f
6	100	1678411781577713774	src/Pages/App/_Document.ast	4758fcc28c9db661
8	102	1678411781579396980	src/Pages/Home/Home.ast	2b4018762967b9cb
52	104	1678411781584024622	bindings/Cypress/Cypress.d	a9554aaf4ea3b4b2
10	124	1678411781602744768	src/Layouts/Default.ast	4cb96a312a38ae78
76	128	1678411781609133729	bindings/Storybook/Storybook.d	c7be9f6a14cae553
13	130	1678411781607261753	src/Components/Navbar/Navber.ast	fad02a4516cb29e5
87	131	1678411781610713655	bindings/Next/Next.d	6e35f5ee90d0d58b
81	132	1678411781612412477	src/Pages/App/_App.d	8b90454047c6cad7
100	153	1678411781633034907	src/Pages/App/_Document.d	9692893aac5aee87
102	155	1678411781635176436	src/Pages/Home/Home.d	11e0519dd80afed0
124	180	1678411781661401816	src/Layouts/Default.d	8f8731417ee5b0cc
104	182	1678411781660745244	bindings/Cypress/Cypress.cmj	8f52204ea9b211e
104	182	1678411781660745244	bindings/Cypress/Cypress.cmi	8f52204ea9b211e
104	182	1678411781660745244	../../bindings/Cypress/Cypress.bs.js	8f52204ea9b211e
130	205	1678411781685540729	src/Components/Navbar/Navber.d	7d1ce9c562396253
131	227	1678411781703341099	bindings/Next/Next.cmj	7afe22d90ff6f1e6
131	227	1678411781703341099	bindings/Next/Next.cmi	7afe22d90ff6f1e6
131	227	1678411781703341099	../../bindings/Next/Next.bs.js	7afe22d90ff6f1e6
129	232	1678411781710908664	bindings/Storybook/Storybook.cmj	4df28b338b09281c
129	232	1678411781710908664	bindings/Storybook/Storybook.cmi	4df28b338b09281c
129	232	1678411781710908664	../../bindings/Storybook/Storybook.bs.js	4df28b338b09281c
132	238	1678411781716633241	src/Pages/App/_App.cmi	c55d50b1460d02a7
153	242	1678411781722122723	src/Pages/App/_Document.cmi	eb8edb0a28d8a279
155	254	1678411781732800833	src/Pages/Home/Home.cmj	e4b6d07e2382fb45
155	254	1678411781732800833	src/Pages/Home/Home.cmi	e4b6d07e2382fb45
155	254	1678411781732800833	../../src/Pages/Home/Home.bs.js	e4b6d07e2382fb45
181	256	1678411781734465640	src/Layouts/Default.cmj	75a3fded494d762b
181	256	1678411781734465640	src/Layouts/Default.cmi	75a3fded494d762b
181	256	1678411781734465640	../../src/Layouts/Default.bs.js	75a3fded494d762b
205	279	1678411781757385810	src/Components/Navbar/Navber.cmj	496fa30e9dba1ec0
205	279	1678411781757385810	src/Components/Navbar/Navber.cmi	496fa30e9dba1ec0
205	279	1678411781757385810	../../src/Components/Navbar/Navber.bs.js	496fa30e9dba1ec0
238	305	1678411781783553719	src/Pages/App/_App.cmj	55eb5d8c6884001a
238	305	1678411781783553719	../../src/Pages/App/_App.bs.js	55eb5d8c6884001a
2	46	1678411799525496368	src/Pages/App/_Document.ast	4758fcc28c9db661
46	73	1678411799525496368	src/Pages/App/_Document.d	9692893aac5aee87
2	47	1678411800976656149	src/Pages/App/_Document.ast	4758fcc28c9db661
47	72	1678411800976656149	src/Pages/App/_Document.d	9692893aac5aee87
1	56	1678411806863997861	src/Pages/App/_Document.ast	4758fcc28c9db661
56	85	1678411806863997861	src/Pages/App/_Document.d	9692893aac5aee87
0	46	1678411781530206192	bindings/Cypress/Cypress.d	8dfe72ddf9b2464c
2	79	1678411781564711302	bindings/Next/Next.d	4621f61e18456f1a
2	94	1678411781579396980	src/Pages/Home/Home.d	d6986eed28ac254
1	100	1678411781553418189	bindings/Storybook/Storybook.d	8c5213c2ef250077
3	121	1678411781602744768	src/Layouts/Default.d	4709b1d94d2829c5
4	131	1678411835591408165	src/App/_App.ast	1188be5018dead3f
4	137	1678411781607261753	src/Components/Navbar/Navber.d	54494a891a169350
5	152	1678411835612263283	src/App/_App.iast	5017fa1eb7851e1e
6	155	1678411835616560506	src/App/_Document.iast	235ef1ab60876c11
6	169	1678411835629444977	src/App/_Document.ast	af0681fa579fd663
47	178	1678411781584024622	bindings/Cypress/Cypress.cmj	ee0ffb80a38e05c6
47	178	1678411781584024622	bindings/Cypress/Cypress.cmi	ee0ffb80a38e05c6
47	178	1678411781584024622	../../bindings/Cypress/Cypress.bs.js	ee0ffb80a38e05c6
94	181	1678411781635176436	src/Pages/Home/Home.cmj	3ec492b23ae6fd9c
94	181	1678411781635176436	src/Pages/Home/Home.cmi	3ec492b23ae6fd9c
94	181	1678411781635176436	../../src/Pages/Home/Home.bs.js	3ec492b23ae6fd9c
101	184	1678411781609133729	bindings/Storybook/Storybook.cmj	6b1acbb9a14fe774
101	184	1678411781609133729	bindings/Storybook/Storybook.cmi	6b1acbb9a14fe774
101	184	1678411781609133729	../../bindings/Storybook/Storybook.bs.js	6b1acbb9a14fe774
152	199	1678411835662161112	src/App/_App.d	35319498229d7dac
79	200	1678411781610713655	bindings/Next/Next.cmj	6d2c23958683b1a9
79	200	1678411781610713655	bindings/Next/Next.cmi	6d2c23958683b1a9
79	200	1678411781610713655	../../bindings/Next/Next.bs.js	6d2c23958683b1a9
122	206	1678411781661401816	src/Layouts/Default.cmj	4946a784c2ac493a
122	206	1678411781661401816	src/Layouts/Default.cmi	4946a784c2ac493a
122	206	1678411781661401816	../../src/Layouts/Default.bs.js	4946a784c2ac493a
137	212	1678411781685540729	src/Components/Navbar/Navber.cmj	d8c226d8175bb3bf
137	212	1678411781685540729	src/Components/Navbar/Navber.cmi	d8c226d8175bb3bf
137	212	1678411781685540729	../../src/Components/Navbar/Navber.bs.js	d8c226d8175bb3bf
169	220	1678411835683934174	src/App/_Document.d	19ae091fb0dd5356
199	267	1678411835728472026	src/App/_App.cmi	d5cba09ec9a19a9c
220	278	1678411835740078536	src/App/_Document.cmi	a6ad71871369a83d
268	325	1678411835786664107	src/App/_App.cmj	86503600f4f598cc
268	325	1678411835786664107	../../src/App/_App.bs.js	86503600f4f598cc
1	60	1678411859161144132	src/App/_Document.ast	af0681fa579fd663
61	95	1678411859161144132	src/App/_Document.d	19ae091fb0dd5356
1	66	1678411866515349386	src/App/_Document.ast	af0681fa579fd663
66	97	1678411866515349386	src/App/_Document.d	19ae091fb0dd5356
1	58	1678411868815053933	src/App/_Document.ast	af0681fa579fd663
58	89	1678411868815053933	src/App/_Document.d	19ae091fb0dd5356
3	96	1678411974377840308	src/Pages/Home/Home.ast	89d4fd42b8549b29
0	101	1678411974383220182	bindings/Cypress/Cypress.ast	1bfbfab1030e290f
2	104	1678411974377634422	bindings/Next/Next.ast	869024cebda8a361
1	107	1678411974382007368	bindings/Storybook/Storybook.ast	ba3c3031990a478f
4	110	1678411974389051039	src/Layouts/Default.ast	f5fe51076ce75f6e
6	112	1678411974387165896	src/App/_App.ast	9eee0858573baaaf
11	112	1678411974392750535	src/App/_Document.ast	1830a1d2bb2a15c0
8	117	1678411974399531258	src/App/_App.iast	30c071b2135bc357
5	119	1678411974399880972	src/Components/Navbar/Navber.ast	c7f8b281d4834726
13	139	1678411974423211988	src/App/_Document.iast	62b129f9d48ea9e9
96	153	1678411974377840308	src/Pages/Home/Home.d	d6986eed28ac254
101	158	1678411974383220182	bindings/Cypress/Cypress.d	8dfe72ddf9b2464c
107	174	1678411974382007368	bindings/Storybook/Storybook.d	8c5213c2ef250077
104	179	1678411974377634422	bindings/Next/Next.d	4621f61e18456f1a
110	202	1678411974389051039	src/Layouts/Default.d	4709b1d94d2829c5
117	205	1678411974399531258	src/App/_App.d	35319498229d7dac
120	208	1678411974399880972	src/Components/Navbar/Navber.d	54494a891a169350
139	218	1678411974423211988	src/App/_Document.d	19ae091fb0dd5356
158	254	1678411974383220182	bindings/Cypress/Cypress.cmj	38627a7ed374894b
158	254	1678411974383220182	bindings/Cypress/Cypress.cmi	38627a7ed374894b
158	254	1678411974383220182	../../bindings/Cypress/Cypress.bs.js	38627a7ed374894b
153	265	1678411974377840308	src/Pages/Home/Home.cmj	3025645b55e6d57d
153	265	1678411974377840308	src/Pages/Home/Home.cmi	3025645b55e6d57d
153	265	1678411974377840308	../../src/Pages/Home/Home.bs.js	3025645b55e6d57d
175	288	1678411974382007368	bindings/Storybook/Storybook.cmj	93ed51892ed0e8f6
175	288	1678411974382007368	bindings/Storybook/Storybook.cmi	93ed51892ed0e8f6
175	288	1678411974382007368	../../bindings/Storybook/Storybook.bs.js	93ed51892ed0e8f6
202	296	1678411974389051039	src/Layouts/Default.cmj	97f4ca605d42a2ae
202	296	1678411974389051039	src/Layouts/Default.cmi	97f4ca605d42a2ae
202	296	1678411974389051039	../../src/Layouts/Default.bs.js	97f4ca605d42a2ae
205	307	1678411974590105342	src/App/_App.cmi	5727d1cdf4ed7ec9
179	312	1678411974377634422	bindings/Next/Next.cmj	f86a02741eb69093
179	312	1678411974377634422	bindings/Next/Next.cmi	f86a02741eb69093
179	312	1678411974377634422	../../bindings/Next/Next.bs.js	f86a02741eb69093
208	316	1678411974399880972	src/Components/Navbar/Navber.cmj	c3cc399f1b723c9
208	316	1678411974399880972	src/Components/Navbar/Navber.cmi	c3cc399f1b723c9
208	316	1678411974399880972	../../src/Components/Navbar/Navber.bs.js	c3cc399f1b723c9
218	322	1678411974604602490	src/App/_Document.cmi	bf66068ba2c3869e
308	412	1678411974590105342	src/App/_App.cmj	14ac5c6ed31a24c5
308	412	1678411974590105342	../../src/App/_App.bs.js	14ac5c6ed31a24c5
322	417	1678411974702420595	src/App/_Document.cmj	c546936afb2bcafc
322	417	1678411974702420595	../../src/App/_Document.bs.js	c546936afb2bcafc
