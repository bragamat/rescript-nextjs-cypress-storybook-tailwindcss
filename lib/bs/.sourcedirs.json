{ "dirs" : [ "bindings" , "bindings/Cypress" , "bindings/Storybook" , "bindings/Next" , "src" , "src/Pages" , "src/Pages/Home" , "src/Layouts" , "src/Components" , "src/Components/Navbar" , "src/App" ] , "pkgs" : [ [ "@rescript/react" , "/Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/@rescript/react" ] ] , "generated" : [] }