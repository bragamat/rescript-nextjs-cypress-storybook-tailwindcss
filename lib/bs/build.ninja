rescript = 1
g_finger := /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/@rescript/react/lib/ocaml/install.stamp
rule astj
  command = /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/rescript/darwin/bsc.exe  -bs-v 10.1.3 -bs-jsx 4 -bs-jsx-mode classic  -absname -bs-ast -o $out $i
o bindings/Cypress/Cypress.ast : astj ../../bindings/Cypress/Cypress.res
rule deps
  command = /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/rescript/darwin/bsb_helper.exe -hash 39f4904b0122083909078819cf529fb6 $in
  restat = 1
o bindings/Cypress/Cypress.d : deps bindings/Cypress/Cypress.ast
rule mij
  command = /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/rescript/darwin/bsc.exe -I src/App -I src/Components/Navbar -I src/Components -I src/Layouts -I src/Pages/Home -I src/Pages -I src -I bindings/Next -I bindings/Storybook -I bindings/Cypress -I bindings -I /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/@rescript/react/lib/ocaml   -bs-package-name rescript-nextjs-cypress-storybook-template -bs-package-output es6:$in_d:.bs.js -bs-v $g_finger $i
  dyndep = 1
  restat = 1
o bindings/Cypress/Cypress.cmj bindings/Cypress/Cypress.cmi ../../bindings/Cypress/Cypress.bs.js : mij bindings/Cypress/Cypress.ast
o bindings/Storybook/Storybook.ast : astj ../../bindings/Storybook/Storybook.res
o bindings/Storybook/Storybook.d : deps bindings/Storybook/Storybook.ast
o bindings/Storybook/Storybook.cmj bindings/Storybook/Storybook.cmi ../../bindings/Storybook/Storybook.bs.js : mij bindings/Storybook/Storybook.ast
o bindings/Next/Next.ast : astj ../../bindings/Next/Next.res
o bindings/Next/Next.d : deps bindings/Next/Next.ast
o bindings/Next/Next.cmj bindings/Next/Next.cmi ../../bindings/Next/Next.bs.js : mij bindings/Next/Next.ast
o src/Pages/Home/Home.ast : astj ../../src/Pages/Home/Home.res
o src/Pages/Home/Home.d : deps src/Pages/Home/Home.ast
o src/Pages/Home/Home.cmj src/Pages/Home/Home.cmi ../../src/Pages/Home/Home.bs.js : mij src/Pages/Home/Home.ast
o src/Layouts/Default.ast : astj ../../src/Layouts/Default.res
o src/Layouts/Default.d : deps src/Layouts/Default.ast
o src/Layouts/Default.cmj src/Layouts/Default.cmi ../../src/Layouts/Default.bs.js : mij src/Layouts/Default.ast
o src/Components/Navbar/Navber.ast : astj ../../src/Components/Navbar/Navber.res
o src/Components/Navbar/Navber.d : deps src/Components/Navbar/Navber.ast
o src/Components/Navbar/Navber.cmj src/Components/Navbar/Navber.cmi ../../src/Components/Navbar/Navber.bs.js : mij src/Components/Navbar/Navber.ast
o src/App/_App.ast : astj ../../src/App/_App.res
o src/App/_App.d : deps src/App/_App.ast src/App/_App.iast
o src/App/_App.iast : astj ../../src/App/_App.resi
rule mi
  command = /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/rescript/darwin/bsc.exe -I src/App -I src/Components/Navbar -I src/Components -I src/Layouts -I src/Pages/Home -I src/Pages -I src -I bindings/Next -I bindings/Storybook -I bindings/Cypress -I bindings -I /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/@rescript/react/lib/ocaml   -bs-v $g_finger $i
  dyndep = 1
  restat = 1
o src/App/_App.cmi : mi src/App/_App.iast
rule mj
  command = /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/rescript/darwin/bsc.exe -bs-read-cmi -I src/App -I src/Components/Navbar -I src/Components -I src/Layouts -I src/Pages/Home -I src/Pages -I src -I bindings/Next -I bindings/Storybook -I bindings/Cypress -I bindings -I /Users/mateusbraga/home/templates/rescript/cypress-storybook-tailwindcss-nextjs/node_modules/@rescript/react/lib/ocaml   -bs-package-name rescript-nextjs-cypress-storybook-template -bs-package-output es6:$in_d:.bs.js -bs-v $g_finger $i
  dyndep = 1
  restat = 1
o src/App/_App.cmj ../../src/App/_App.bs.js : mj src/App/_App.ast src/App/_App.cmi
o src/App/_Document.ast : astj ../../src/App/_Document.res
o src/App/_Document.d : deps src/App/_Document.ast src/App/_Document.iast
o src/App/_Document.iast : astj ../../src/App/_Document.resi
o src/App/_Document.cmi : mi src/App/_Document.iast
o src/App/_Document.cmj ../../src/App/_Document.bs.js : mj src/App/_Document.ast src/App/_Document.cmi
