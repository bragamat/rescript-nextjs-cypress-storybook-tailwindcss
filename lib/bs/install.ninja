rescript = 1
rule cp
  command = cp $i $out
rule touch
 command = touch $out
o Cypress.cmi : cp ../bs/bindings/Cypress/Cypress.cmi
o Cypress.cmj : cp ../bs/bindings/Cypress/Cypress.cmj
o Cypress.cmt : cp ../bs/bindings/Cypress/Cypress.cmt
o Cypress.res : cp ../../bindings/Cypress/Cypress.res
o Storybook.cmi : cp ../bs/bindings/Storybook/Storybook.cmi
o Storybook.cmj : cp ../bs/bindings/Storybook/Storybook.cmj
o Storybook.cmt : cp ../bs/bindings/Storybook/Storybook.cmt
o Storybook.res : cp ../../bindings/Storybook/Storybook.res
o Next.cmi : cp ../bs/bindings/Next/Next.cmi
o Next.cmj : cp ../bs/bindings/Next/Next.cmj
o Next.cmt : cp ../bs/bindings/Next/Next.cmt
o Next.res : cp ../../bindings/Next/Next.res
o Home.cmi : cp ../bs/src/Pages/Home/Home.cmi
o Home.cmj : cp ../bs/src/Pages/Home/Home.cmj
o Home.cmt : cp ../bs/src/Pages/Home/Home.cmt
o Home.res : cp ../../src/Pages/Home/Home.res
o Default.cmi : cp ../bs/src/Layouts/Default.cmi
o Default.cmj : cp ../bs/src/Layouts/Default.cmj
o Default.cmt : cp ../bs/src/Layouts/Default.cmt
o Default.res : cp ../../src/Layouts/Default.res
o Navber.cmi : cp ../bs/src/Components/Navbar/Navber.cmi
o Navber.cmj : cp ../bs/src/Components/Navbar/Navber.cmj
o Navber.cmt : cp ../bs/src/Components/Navbar/Navber.cmt
o Navber.res : cp ../../src/Components/Navbar/Navber.res
o _App.cmi : cp ../bs/src/App/_App.cmi
o _App.cmj : cp ../bs/src/App/_App.cmj
o _App.cmt : cp ../bs/src/App/_App.cmt
o _App.res : cp ../../src/App/_App.res
o _App.resi : cp ../../src/App/_App.resi
o _App.cmti : cp ../bs/src/App/_App.cmti
o _Document.cmi : cp ../bs/src/App/_Document.cmi
o _Document.cmj : cp ../bs/src/App/_Document.cmj
o _Document.cmt : cp ../bs/src/App/_Document.cmt
o _Document.res : cp ../../src/App/_Document.res
o _Document.resi : cp ../../src/App/_Document.resi
o _Document.cmti : cp ../bs/src/App/_Document.cmti
build install.stamp : touch Cypress.cmi Cypress.cmj Storybook.cmi Storybook.cmj Next.cmi Next.cmj Home.cmi Home.cmj Default.cmi Default.cmj Navber.cmi Navber.cmj _App.cmi _App.cmj _Document.cmi _Document.cmj 
