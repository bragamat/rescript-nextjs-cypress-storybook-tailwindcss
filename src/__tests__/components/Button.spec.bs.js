// Generated by ReScript, PLEASE EDIT WITH CARE

import * as React from "react";
import * as Vitest from "rescript-vitest/src/Vitest.bs.js";
import * as Vitest$1 from "vitest";
import * as ReactTestingLibrary from "rescript-react-testing-library/src/ReactTestingLibrary.bs.js";

function Button$dotspec$DummyComponent(props) {
  return React.createElement("div", undefined, React.createElement("label", undefined, "Choose a color"), React.createElement("select", undefined, React.createElement("option", undefined, "Red"), React.createElement("option", undefined, "Green"), React.createElement("option", {
                      onClick: (function (param) {
                          console.log("Blue");
                        })
                    }, "Blue")));
}

var DummyComponent = {
  make: Button$dotspec$DummyComponent
};

Vitest.describe("DummyComponent", undefined, (function (param) {
        Vitest$1.beforeEach(function () {
              return ReactTestingLibrary.renderOnScreen(React.createElement(Button$dotspec$DummyComponent, {}));
            });
        Vitest.test("render Red option", undefined, (function (param) {
                Vitest$1.expect(ReactTestingLibrary.getByRole(ReactTestingLibrary.makeByRoleOptions(undefined, undefined, "Red", undefined), {
                            NAME: "Str",
                            VAL: "option"
                          }, document)).toMatchSnapshot();
              }));
        Vitest.test("render Green option", undefined, (function (param) {
                Vitest$1.expect(ReactTestingLibrary.getByRole(ReactTestingLibrary.makeByRoleOptions(undefined, undefined, "Green", undefined), {
                            NAME: "Str",
                            VAL: "option"
                          }, document)).toMatchSnapshot();
              }));
        Vitest.test("render Blue option", undefined, (function (param) {
                Vitest$1.expect(ReactTestingLibrary.getByRole(ReactTestingLibrary.makeByRoleOptions(undefined, undefined, "Blue", undefined), {
                            NAME: "Str",
                            VAL: "option"
                          }, document)).toMatchSnapshot();
              }));
      }));

export {
  DummyComponent ,
}
/*  Not a pure module */
