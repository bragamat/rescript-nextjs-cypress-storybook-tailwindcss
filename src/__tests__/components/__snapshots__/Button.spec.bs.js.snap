// Vitest Snapshot v1

exports[`DummyComponent > render Blue option 1`] = `
<option>
  Blue
</option>
`;

exports[`DummyComponent > render Green option 1`] = `
<option>
  Green
</option>
`;

exports[`DummyComponent > render Red option 1`] = `
<option>
  Red
</option>
`;
